### ISSUE TITLE
I am unable to compile with SBCL 2.1.9 on macOS?

### ISSUE THREAD

**Issue by Dima-369** [melisgl/try#1]
This is the error I get when cloning the current `main` branch and running `(ql:quickload 'try)`:

```lisp
; file: /Users/Gira/try/src/result.lisp
; in: define-combi-event (expected result success)
;     (TRY::DEFINE-COMBI-EVENT (TRY:EXPECTED TRY:RESULT TRY:SUCCESS))
; ==>
;   (DEFINE-CONDITION TRY::|expected-result-success|
;       (TRY::|expected| TRY::|result| TRY::|success|)
;       NIL)
; 
; caught error:
;   (during macroexpansion of (define-condition |expected-result-success|
;       ...))
;   Class not yet defined: |expected|

; in: define-combi-event (unexpected result success)
;     (TRY::DEFINE-COMBI-EVENT (TRY:UNEXPECTED TRY:RESULT TRY:SUCCESS))
; ==>
;   (DEFINE-CONDITION TRY::|unexpected-result-success|
;       (TRY::|unexpected| TRY::|result| TRY::|success|)
;       NIL)
; 
; caught error:
;   (during macroexpansion of (define-condition |unexpected-result-success|
;       ...))
;   Class not yet defined: |unexpected|

; in: define-combi-event (expected result failure)
;     (TRY::DEFINE-COMBI-EVENT (TRY:EXPECTED TRY:RESULT TRY:FAILURE))
; ==>
;   (DEFINE-CONDITION TRY::|expected-result-failure|
;       (TRY::|expected| TRY::|result| TRY::|failure|)
;       NIL)
; 
; caught error:
;   (during macroexpansion of (define-condition |expected-result-failure|
;       ...))
;   Class not yet defined: |expected|

; in: define-combi-event (unexpected result failure)
;     (TRY::DEFINE-COMBI-EVENT (TRY:UNEXPECTED TRY:RESULT TRY:FAILURE))
; ==>
;   (DEFINE-CONDITION TRY::|unexpected-result-failure|
;       (TRY::|unexpected| TRY::|result| TRY::|failure|)
;       NIL)
; 
; caught error:
;   (during macroexpansion of (define-condition |unexpected-result-failure|
;       ...))
;   Class not yet defined: |unexpected|

; in: define-combi-event (result skip)
;     (TRY::DEFINE-COMBI-EVENT (TRY:RESULT TRY:SKIP))
; ==>
;   (DEFINE-CONDITION TRY::|result-skip|
;       (TRY::|result| TRY::|skip|)
;       NIL)
; 
; caught error:
;   (during macroexpansion of (define-condition |result-skip|
;       ...))
;   Class not yet defined: |result|

; in: define-combi-event (result success)
;     (TRY::DEFINE-COMBI-EVENT (TRY:RESULT TRY:SUCCESS))
; ==>
;   (DEFINE-CONDITION TRY::|result-success|
;       (TRY::|result| TRY::|success|)
;       NIL)
; 
; caught error:
;   (during macroexpansion of (define-condition |result-success|
;       ...))
;   Class not yet defined: |result|

; in: define-combi-event (result failure)
;     (TRY::DEFINE-COMBI-EVENT (TRY:RESULT TRY:FAILURE))
; ==>
;   (DEFINE-CONDITION TRY::|result-failure|
;       (TRY::|result| TRY::|failure|)
;       NIL)
; 
; caught error:
;   (during macroexpansion of (define-condition |result-failure|
;       ...))
;   Class not yet defined: |result|
..
; 
; compilation unit finished
;   caught 7 ERROR conditions
; Debugger entered on #<uiop/lisp-build:compile-file-error {100527EF53}>
```

**Comment by melisgl** [comment]
What do the following evaluate to?

```
*print-case*
(readtable-case *readtable*)
(readtable-case (named-readtables:find-readtable 'pythonic-string-reader:pythonic-string-syntax))
```

**Comment by Dima-369** [comment]
![image](https://user-images.githubusercontent.com/15002298/156118345-5cd86d3e-3553-4b8f-acca-c9f16f2e65c7.png)

**Comment by Dima-369** [comment]
Aha, if I use `(setq *print-case* :upcase)` , `(ql:quickload 'try)` works!

**Comment by melisgl** [comment]
Thank you for the report. Fixed in a1fffad.
